unit-tests/shared/src/test/scala/javalib/util/AbstractMapTest.scala

```diff
@@ -1,4 +1,4 @@
-package javalib.util
+package org.scalanative.testsuite.javalib.util
```

Comment:
I think I was thinking it made it closer to the Scala.js packages and I thought there were some others like that too. I think since this would involve moving the code, sticking with the current package scheme is better.

Reply:
Ok, looking at it - bunches of tests have the package starting with `org.scalanative.testsuite` but none in the matching package structure. I started changing back but realize that all the new ported code is in the new package. I think that is why I put the new stuff in the new package. We could put in an issue to change the code in the direction we would like to go unless you would like me to change the other code too but there are lots of files.

Reply:
I can move the files into the new directory structure - the advantage of that is it shows which code and tests are newer or more recently ported from Scala.js. It seems we started down that path but never moved the files.

Reply:
In such case, we can leave it as it is now, and maybe fix the layout in the future

Reply:
I will put in a issue because I think it it better to use the new package.

Reply:
Added: https://github.com/scala-native/scala-native/issues/2432